executor/executor_test.go

```diff
@@ -1850,6 +1850,12 @@ func (s *testSuiteP1) TestGeneratedColumnRead(c *C) {
 	result = tk.MustQuery(`SELECT * FROM test_gc_read_m`)
 	result.Check(testkit.Rows(`1 2 4`, `2 3 6`))
 
+	// Test for issue #12233
```

Comment:
Please do not put it in `TestGeneratedColumnRead`

Reply:
thank you. I had removed this comment.

Reply:
Sorry, maybe you don't get me.

Your test case is in `TestGeneratedColumnRead`, but it should not be here.